.github/workflows/installer-macos.yml

```diff
@@ -21,37 +21,42 @@ jobs:
       LITE_FLAG: ${{ matrix.build_type == 'Lite' && '--lite' || '' }}
       DMGNAME: ${{ matrix.build_type == 'Lite' && 'Spyder-Lite.dmg' || 'Spyder.dmg' }}
       DISTDIR: ${{ github.workspace }}/dist
+      MACOSX_DEPLOYMENT_TARGET: 10.9
     steps:
       - name: Checkout Code
         uses: actions/checkout@v2
         with:
           fetch-depth: 0
       - name: Update System
         run: |
+          python3.9 -c "import sysconfig; print(f'''MACOSX_DEPLOYMENT_TARGET: {sysconfig.get_config_var('MACOSX_DEPLOYMENT_TARGET')}''')"
+          brew uninstall --force --ignore-dependencies python
           brew install spatialindex
```

Comment:
You need to make sure this library is compiled for the new `MACOSX_DEPLOYMENT_TARGET` and not installed from Homebrew Catalina's bottle (that's the name for binary packages in Homebrew). Otherwise things will fail too (I think).

Unfortunately, this is necessary because the new `rtree` wheels are only compatible with Catalina.

Reply:
From the logs that doesn't seem to be the case.

Reply:
I don't know if you saw my latest commit...I deliberately installed Python from python.org for macOS 10.9. At the moment, we're still using the wheel-less rtree (they haven't updated on the conda default channel yet). But that is something to consider in the future. We may end up having to do multiple platform builds...
Nevertheless, I was hoping to see if this at least launched correctly in Mojave, before tackling any subsequent issues.

Reply:
> I don't know if you saw my latest commit...I deliberately installed Python from python.org for macOS 10.9

@mrclary, yes, I saw that you're using a build from python.org now. My comment was meant to let you know that we can't use packages from `spatialindex` in the installer if we want to make it compatible with Mojave, because Homebrew is installing a Catalina bottle. And that will (probably) make the installer crash too.

> Nevertheless, I was hoping to see if this at least launched correctly in Mojave, before tackling any subsequent issues.

Ok, I have a Mojave VM so I can help you with that,

Reply:
I've just pushed up changes to use mojave bottles for both python and spatialindex, and builds the full version. Let's see if that works...